src/main/java/Duke.java

```diff
@@ -1,3 +1,54 @@
+import java.util.Scanner;
+public class Duke {
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+    public static void main(String[] args) {
+        String[] list = new String[100];
+        String[] status = new String[100];
+        //initializing the array
+        for (int i = 0; i < 100; i++) {
+            list[i] = " ";
+            status[i] = " ";
+        }
+        int index = 0;
+        Scanner in = new Scanner(System.in);
+        printLine();
+        System.out.println("Hello! I'm Doraemon!");
+        System.out.println("What can I do for you?");
+        printLine();
+        while (!list[index].equals("bye")) {
+            list[index] = in.nextLine();
+            if (list[index].equals("bye")) {
```

Comment:
Can consider abstracting the list[index].equals logic to another class in conjunction with using switch case statements to improve readibility